src/eth_provider/constant.rs

```diff
@@ -59,3 +60,17 @@ lazy_static! {
         );
     pub static ref DEPLOY_WALLET_NONCE: Arc<Mutex<Felt>> = Arc::new(Mutex::new(Felt::ZERO));
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+// Struct used to return the constant values from the kakarot_getConfig endpoint
+pub struct Constant {
+    pub max_priority_fee_per_gas: u64,
+    pub max_logs: Option<u64>,
+    pub call_request_gas_limit: u128,
+    pub hash_hex_string_len: usize,
+    pub logs_topics_hex_string_len: usize,
+    pub u64_hex_string_len: usize,
+    pub block_number_hex_string_len: usize,
+    pub address_hex_string_len: usize,
+    pub starknet_modulus: U256,
```

Comment:
we can remove all values that don't depend on `var`, so let's keep/add:
- MAX_LOGS
- STARKNET_NETWORK
- RETRY_TX_INTERVAL
- TRANSACTION_MAX_RETRIES
- MAX_FELTS_IN_CALLDATA
- WHITE_LISTED_EIP_155_TRANSACTION_HASHES

Reply:
The rest are all hardcoded for now, so we don't really need to query them at runtime

Reply:
fixed, thanks